### PR TITLE
[ticket/13790] Update phpcs

### PR DESCRIPTION
**Pull request by Nicofuma**
https://tracker.phpbb.com/browse/PHPBB3-13790

PHPBB3-13790

### CODE REVIEW

**Comment by Nicofuma** [comment]
The errors are not related to this PR (relevant lines: https://travis-ci.org/phpbb/phpbb/jobs/60300443#L576-L605)

File: build/code_sniffer/phpbb/Sniffs/Commenting/FileCommentSniff.php [hunk 1]
```diff
@@ -176,17 +168,23 @@ protected function processVersion(PHP_CodeSniffer_File $phpcsFile, $ptr, $tags)
...
-		if (!isset($tags['copyright']))
-		{
-			$message = 'Missing require @copyright tag in file doc comment.';
-			$phpcsFile->addError($message, $ptr);
-		}
-		else if ($tags['copyright'][0][0] !== $copyright)
-		{
-			$message = 'Invalid content found for the first @copyright tag, use "' . $copyright . '".';
-			$phpcsFile->addError($message, $tags['copyright'][0][1]);
+		foreach ($tags as $tag) {
```
Comment (bantu) [inline build/code_sniffer/phpbb/Sniffs/Commenting/FileCommentSniff.php]: newline

File: build/code_sniffer/phpbb/Sniffs/Commenting/FileCommentSniff.php [hunk 2]
```diff
@@ -201,7 +199,7 @@ protected function processCopyright(PHP_CodeSniffer_File $phpcsFile, $ptr, $tags
 	protected function processLicense(PHP_CodeSniffer_File $phpcsFile, $ptr, $tags)
 	{
 		$license = 'GNU General Public License, version 2 (GPL-2.0)';
-
+/*
```
Comment (bantu) [inline build/code_sniffer/phpbb/Sniffs/Commenting/FileCommentSniff.php]: ?

File: phpBB/composer.json [hunk 3]
```diff
@@ -25,8 +25,9 @@
 		"phpbb/phpbb-core": "self.version"
 	},
 	"require": {
-		"php": ">=5.3.3",
 		"lusitanian/oauth": "0.2.*",
+		"php": ">=5.3.3",
+		"squizlabs/php_codesniffer": "2.*",
```
Comment (bantu) [inline phpBB/composer.json]: Why require?

**Comment by Nicofuma** [comment]
updated

File: build/code_sniffer/phpbb/Sniffs/Namespaces/UnusedUseSniff.php [hunk 4]
```diff
@@ -147,43 +148,35 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
...
+							$ok = $this->check($phpcsFile, $class, $class_name_full, $class_name_short, $tokens[$tag + 2]['line']) ? true : $ok;
 						}
 					}
-					catch (PHP_CodeSniffer_CommentParser_ParserException $e)
-					{
-						$line = ($e->getLineWithinComment() + $comment_start);
-						$phpcsFile->addError($e->getMessage(), $line, 'FailedParse');
-					}
+				}
+				else// if ($tokens[$comment_end]['code'] === T_DOC_COMMENT)
```
Comment (bantu) [inline build/code_sniffer/phpbb/Sniffs/Namespaces/UnusedUseSniff.php]: ?

Comment (Nicofuma) [inline build/code_sniffer/phpbb/Sniffs/Namespaces/UnusedUseSniff.php]: updated

**Pull request merged**